code/style.css

```diff
@@ -1,58 +1,377 @@
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 100px;
+}
+
+.header-text {
+  font-size: 6.5em;
+  color: crimson;
+  margin-bottom: -50px;
+}
+
+/*** DRIPPING ANIMATION ***/
+
+.drop {
+  position: relative;
+  width: 20px;
+  height: 20px;
+  top: 5px;
+  margin: 0 0 0 360px;
+  background: crimson;
+  -moz-border-radius: 20px;
+  -webkit-border-radius: 20px;
+  border-radius: 20px;
+  -moz-animation: drip 3s cubic-bezier(1, 0, 0.91, 0.19) infinite;
+  -webkit-animation: drip 3s cubic-bezier(1, 0, 0.91, 0.19) infinite;
+  animation: drip 3s cubic-bezier(1, 0, 0.91, 0.19) infinite;
+}
+
+.drop:before {
+  content: '';
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-bottom: 30px solid crimson;
+  top: -22px;
+}
+
+/*** HEADER BUTTON ***/
+
+.button-start {
+  color: #fff;
+  -moz-animation: pulse 5s infinite;
+  -webkit-animation: pulse 5s infinite;
+  animation: pulse 5s infinite;
+  cursor: pointer;
+  transition: transform 0.2s;
+}
+
+.button-start:hover {
+  transform: scale(1.2);
+  animation: none;
+}
+
+/*** GAMEBOARD WINDOW ***/
+
+.game-wrapper {
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  margin: 0 auto;
+  text-align: center;
+}
+
+/*** CARDS ***/
+
+.cards-container {
   display: flex;
-  justify-content: space-around;
+  justify-content: space-evenly;
   align-items: center;
+  flex-direction: column;
+  max-width: 1100px;
+  margin: auto;
+  padding: 16px;
 }
 
-.killer-deck {
+.info-header {
+  color: #fff;
+  margin-top: 50px;
+  margin: auto;
+  padding: 20px;
+}
+
+.info {
+  color: #fff;
+  background: none;
+}
+
+.decks {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   width: 250px;
   height: 400px;
-  background: #000;
   text-align: center;
   padding-top: 30px;
+  padding-bottom: 20px;
   border-radius: 16px;
+  cursor: pointer;
+  background: #000;
+  margin: 25px 0;
 }
 
-.weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
+.killer-deck {
+  position: relative;
+}
+
+.killer-image {
+  margin: 15px 0;
+  border-radius: 50%;
+}
+
+.p-header {
+  margin-bottom: 20px;
+  background-color: crimson;
+  color: #fff;
+  font-weight: bold;
+  letter-spacing: 2px;
+}
+
+/*** LOADER ***/
+
+.loader-container {
+  position: absolute;
+  margin-top: 100px;
+  width: 70px;
+  height: 70px;
+}
+
+.loader-container-animation {
+  -webkit-animation: scale-spin 1.5s linear infinite;
+  -moz-animation: scale-spin 1.5s linear infinite;
+  animation: scale-spin 1.5s linear infinite;
+}
+
+.loader-image {
+  width: 100%;
+}
+
+/*** REVEAL WINDOW ***/
+
+.reveal-wrapper {
+  flex-direction: column;
+  align-items: center;
+  width: 90%;
+  height: 100vh;
+  margin: 0 auto;
   text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
```

Comment:
Maybe I should ask myself instead "Do I need to have display: flex to make flex-direction: column work?" Huhm...I need to research more on this.